content/06.discussion.md

```diff
@@ -269,13 +269,42 @@ comparable performance. Towards this end, Che et al. [@tag:Che2015_distill] used
 gradient boosted trees to learn interpretable healthcare features from trained
 deep models.
 
-Finally, it is sometimes possible to train the model to provide justifications
+It is sometimes possible to train the model to provide justifications
 for its predictions. Lei et al. [@tag:Lei2016_rationalizing] used a generator to
 identify "rationales", which are short and coherent pieces of the input text
 that produce similar results to the whole input when passed through an encoder.
 The authors applied their approach to a sentiment analysis task and obtained
 substantially superior results compared to an attention-based method.
 
+Lastly, interpretation of embedded or latent space features learned through
+generative unsupervised models can reveal underlying patterns otherwise masked
+in the original input. Embedded feature interpretation has been used mostly in
+image and text based applications [@tag:Radford_dcgan; @tag:Word2Vec], but
+applications to genomic and biomedical domains are increasing. For example, Way
+and Greene train a variational autoencoder (VAE) on gene expression from The
+Cancer Genome Atlas (TCGA) and use latent space arithmetic to rapidly isolate
+and interpret gene expression features descriptive of two high grade serous
+ovarian cancer subtypes [@tag:WayGreene2017_tybalt]. The most differentiating
+VAE features were representative of biological processes that are known to
+distinguish the subtypes. Latent space arithmetic with features derived using
+other compression algorithms were not as informative in this context
+[@tag:WayGreene2017_eval]. Embedding discrete chemical structures with
+autoencoders and interpreting the learned continuous representations with latent
+space arithmetic has also facilitated the discovery of drug compounds
+[@tag:Gomezb2016_automatic]. Furthermore, embedding biomedical text
+into lower dimensional latent spaces have improved state of the art name entity
+recognition in a variety of tasks including annotating clinical abbreviations,
+genes, cell lines, and drug names [@doi:10.3390/info6040848;
+@doi:10.1155/2014/240403; @doi:10.18653/v1/w15-3822; @doi:10.18653/v1/w15-3810].
+Other approaches have interpolated through latent space embeddings learned by
+generative adversarial networks (GAN) to interpret unobserved intermediate
```

Comment:
`GANs` has been defined at the end of the treat section

Reply:
:+1: